src/Command/TemplateCommand.php

```diff
@@ -268,6 +268,8 @@ protected function _loadController(ConsoleIo $io): array
 
         $namespace = Configure::read('App.namespace');
 
+        $primaryKey = $displayField = $singularVar = $singularHumanName = null;
+        $schema = $fields = $modelClass = null;
```

Comment:
`$this->abort();` will throws the `StopException` in catch block 
why phpstan need these then ?

Reply:
Because it is buggy i think.